src/components/Card/Card.js

```diff
@@ -22,11 +22,12 @@ const channelsList = channels =>
         href={url}
         target="_blank"
         rel="noopener noreferrer"
+        class="channel-label"
```

Comment:
`typo` here? @emmawedekind

Reply:
YES className I believe... React and all :D